plugins/inputs/jti_openconfig_telemetry/README.md

```diff
@@ -0,0 +1,46 @@
+# JTI OpenConfig Telemetry Input Plugin
+
+This plugin reads Juniper Networks implementation of OpenConfig telemetry data from listed sensors using Junos Telemetry Interface. Refer to
+[openconfig.net](http://openconfig.net/) for more details about OpenConfig and [Junos Telemetry Interface (JTI)](https://www.juniper.net/documentation/en_US/junos/topics/concept/junos-telemetry-interface-oveview.html).
+
+### Configuration:
+
+```toml
+# Subscribe and receive OpenConfig Telemetry data using JTI
+[[inputs.jti_openconfig_telemetry]]
+  server = ["localhost:1883"]
+
+  ## Frequency to get data in milliseconds
+  sampleFrequency = 2000
+
+  ## Sensors to subscribe for
+  ## A identifier for each sensor can be provided in path by separating with space
+  ## Else sensor path will be used as identifier. If a integer is provided before 
+  ## sensor path, it will be used as reporting rate for that sensor instead of global
+  ## reporting rate
+  sensors = [
+   "/interfaces/",
+   "collection /components/ /lldp",
+  ]
```

Comment:
Add example of custom reporting rate, and explain how the collection works.

Reply:
Done!